delete_old/delete.py

```diff
@@ -0,0 +1,98 @@
+"""deletes old out of date data from the database"""
```

Comment:
Can we make this a proper sentence, also full stops across the board on doc strings